insights_client/__init__.py

```diff
@@ -30,13 +31,16 @@
 try:
     insights_uid = pwd.getpwnam("insights").pw_uid
     insights_gid = pwd.getpwnam("insights").pw_gid
+    insights_grpid = grp.getgrpname("insights").gr_gid
```

Comment:
Is this always the same as `pwd.getpwnam("insights").pw_gid`?

Reply:
I don't think so? I would feel safer if we explicitly got the group ID. But you might be correct.

Reply:
Yeah that's fine.